src/process/message_processor.go

```diff
@@ -70,8 +60,8 @@ func ProcessBGPMessages(msgChannel chan common.Message, config *config.Configura
 			}
 
 			// If we at least maximumBuckets in bucketMap we can perform analysis
-			if len(currWindow.BucketMap) >= maximumBuckets {
-				fmt.Printf("len(currWindow.BucketMap): %d maximumBuckets: %d\n", len(currWindow.BucketMap), maximumBuckets)
+			if len(currWindow.BucketMap) >= config.MaxBuckets {
+				fmt.Println(fmt.Sprintf("len(currWindow.BucketMap): %d maximumBuckets: %d", len(currWindow.BucketMap), config.MaxBuckets))
```

Comment:
You could do this: 

`fmt.Printf("Bucket Map Length: %d: Maximum Buckets %d\n", len(currWindow.Bucketmap), config.MaxBuckets)`